python/cuml/test/dask/test_random_forest.py

```diff
@@ -180,9 +137,7 @@ def test_rf_regression_dask_fil(partitions_per_worker, cluster):
         X_train_df = \
             dask_cudf.from_cudf(X_cudf, npartitions=n_partitions)
 
-        y_cudf = np.array(pd.DataFrame(y_train).values)
-        y_cudf = y_cudf[:, 0]
-        y_cudf = cudf.Series(y_cudf)
+        y_cudf = cudf.Series(y_train)
         y_train_df = \
             dask_cudf.from_cudf(y_cudf, npartitions=n_partitions)
         X_cudf_test = cudf.DataFrame.from_pandas(pd.DataFrame(X_test))
```

Comment:
The test `test_rf_classification_dask_fil` and `test_rf_classification_dask_array` can be combined to a single test.